solr/core/src/test/org/apache/solr/search/facet/TestJsonFacets.java

```diff
@@ -4907,6 +4907,38 @@ public void testQueryJoinBooksAndPages() throws Exception {
             + ", books2:{ buckets:[ {val:q,count:1}, {val:w,count:1} ] }"
             + "}");
   }
+  
+  @Test
+  public void testMultivalueEnumTypes() throws Exception {
+    final Client client = Client.localClient();
+    
+    final SolrParams p = params("rows", "0");
+    
+    client.deleteByQuery("*:*", null);
+    
+    List<SolrInputDocument> docsToAdd = new ArrayList<>(6);
```

Comment:
Can you add them to an UpdateRequest so they are batched instead of sending them individually?  Same LOC in the end but avoids an anti-pattern.  Or is the separation pertinent to what's being tested?

Reply:
I tried but this will make the test more complicated as SolrTestCaseHS.Client does not keep a SolrClient and uses null therefore in internal API calls, may you have an hint for that, but the whole test class is not using an UpdateRequest